### ISSUE TITLE
Save posts by sending them to yourself

### ISSUE THREAD

**Issue by mxsinister** [bluesky-social/social-app#5075]
There have been many requests and workarounds for a way to save posts, i've thought of a simple solution. Just let the user send it to themselves on the "Send via direct message." option.

**Comment by Sininini** [comment]
That’s a clever, albeit odd idea. I think they really should just implement real bookmarks. With folders.

**Comment by wtlgo** [comment]
I agree, they should create proper bookmarks, but it would require a significant amount of time and effort. On the other hand, theoretically, implementing a 'message-to-self' feature shouldn't be as difficult and could serve as a quick workaround for bookmark functionality before the full feature is introduced. I believe the developers did something similar with GIFs earlier this year before fully supporting them. Additionally, almost all messaging apps allow users to send messages to themselves, so it's a feature that many would expect.

By the way, I found this thread because I had the same suggestion. :^)

**Comment by tannal** [comment]
It is also useful when you need to transfer text/files across different devices.
